frontend/src/CardGallery/components/PokemonCards/PokemonCard.tsx

```diff
@@ -1,28 +1,48 @@
-import {Button, Card, CardActionArea, CardActions, CardContent, CardMedia, Grid, Typography} from "@mui/material";
+import {
+    Button,
+    Card,
+    CardActionArea,
+    CardActions,
+    CardContent,
+    CardMedia, Checkbox,
+    Grid,
+    Typography
+} from "@mui/material";
 import {PokeCard} from "../../models/PokeCard";
 import {useNavigate} from "react-router-dom";
+import {Favorite, FavoriteBorder} from "@mui/icons-material";
+
 
 
 type PokemonCardProps = {
     card: PokeCard
     addCardToFavorites(id:string):void
+    isCardInFavorites:(cardId: string) => boolean
+    removeCardFromFavorites: (cardId: string) => Promise<void>;
 
 }
 
 export default function PokemonCard(props: PokemonCardProps) {
 
 
+
+    const handleChange = () => {
```

Comment:
Hier vlt
- den selben Namen wie in der anderen Komoponente nehmen
- und den selben Syntaxstil wie im Rest des Projekts verwenden (warum nicht `function`?)